lambdas/archive-vcs/src/main/java/uk/gov/di/ipv/core/archivevcs/domain/RevokeRequest.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.di.ipv.core.archivevcs.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RevokeRequest {
+    private List<String> userIds;
```

Comment:
You might want to make these `userId` + `criId` pairs to make it a bit more flexible in future?